src/common/containers/Header.jsx

```diff
@@ -11,20 +12,29 @@ export default class Header extends Component {
 
         this.state = {
             user: {},
+            userIsAuth: false,
         };
+        this.onClick = this.onClick.bind(this);
     }
 
     componentDidMount() {
         API.whoAmI().then((response) => {
             this.setState(() => ({
                 user: response.data,
+                userIsAuth: response.status === 200,
             }));
         });
     }
 
+    static onClick(event) {
+        event.preventDefault();
+        signout();
+    }
+
     render() {
         const { user } = this.state;
         const { logo, pushmenu, styleName } = this.props;
+        const { userIsAuth } = this.state;
```

Comment:
You can combine this string with `https://github.com/tolstoyevsky/cusdeb-web-client/pull/8/files#diff-2a6ece80dd273c0b7270daecc93b0278R35`